modules/nodes-nodes-audit-log-basic.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-pods-audit-log-basic-{context}"]
+= About the Basic Audit Log
```

Comment:
We should clarify if this is basic or advance related. It seems more tied to advanced.